pkg/metrics/grafana/overview.json

```diff
@@ -253,6 +253,8 @@
               "refId": "M"
             }
           ],
+          "title": "Probe Success",
```

Comment:
this and below panel don't have a `title`, not sure what's it used for, I add a temporary title based on the expression